demisto_sdk/commands/prepare_content/integration_script_unifier.py

```diff
@@ -552,10 +552,11 @@ def insert_pack_version(
             existing_pack_info_debug_log_re = (
                 r"demisto\.debug\('pack name = .*?, pack version = .*?'\)"
             )
-            global_name = f"{pack_id.upper()}_PACK_VERSION"
+            global_name = "CONSTANT_PACK_VERSION"
+            pack_id_escaped = pack_id.replace("'", "\\'")
             pack_version_statement = (
                 f"{global_name} = '{pack_version}'"
-                f"\ndemisto.debug(f'pack id = {pack_id}, pack version = {{{global_name}}}')"
+                f"\ndemisto.debug('pack id = {pack_id_escaped}, pack version = {pack_version}')"
```

Comment:
you need to use format here. Also why are you no longer using the global name?

Reply:
To avoid problems, I thought it might be better to write straight